test/api-test.js

```diff
@@ -995,6 +995,12 @@ describe('RippleAPI', function() {
         responses.getTrustlines.filtered, 'getTrustlines'));
   });
 
+  it('getTrustlines - more than 400 items', function() {
+    const options = {limit: 401};
```

Comment:
Only asking, but do we need to set the `marker` in the options for the request? If so, should we have multiple tests for different markers?

Reply:
No, I think `marker` is just handled internally by ripple-lib. See [the parameters for `getTrustlines` `options`](https://ripple.com/build/rippleapi/#gettrustlines).